components/InteriorLeftNavItem.js

```diff
@@ -8,8 +8,8 @@ const propTypes = {
   activeHref: PropTypes.string,
   tabIndex: PropTypes.number,
   onClick: PropTypes.func,
-  label: PropTypes.string.isRequired,
   blankTarget: PropTypes.bool,
+  children: PropTypes.node,
```

Comment:
Nevermind. I just realized that `children` is a special case and everything I've been saying here is irrelevant for it. 🤷‍♀️